ucfcs/GrowthPlus
# Julio/realm bugs

This PR implements a realm listener for parent sign up to fix the random nullPointerException bug  and implements onDestroy() methods for all activities. 

To test: 
- Create a new emulator device in Android Studio, it seems like the bug happens on a brand new phone with no data. 
- Create a parent like usual, it should take you to login without causing a null pointer exception. 
- Navigate around the app and try stuff, make sure that nothing breaks.
